sdk/storage/azure-storage-file-share/azure/storage/fileshare/_generated/operations/_share_operations.py

```diff
@@ -395,8 +395,7 @@ def get_permission(self, file_permission_key=None, timeout=None, cls=None, **kwa
         # Construct headers
         header_parameters = {}
         header_parameters['Accept'] = 'application/json'
-        if file_permission_key is not None:
-            header_parameters['x-ms-file-permission-key'] = self._serialize.header("file_permission_key", file_permission_key, 'str')
+        header_parameters['x-ms-file-permission-key'] = self._serialize.header("file_permission_key", file_permission_key, 'str')
```

Comment:
This means permissions is now required, did you check this doesn't break anything and our surface API already was making this required?

Reply:
yes, it's required.